python/mxnet/gluon/rnn/rnn_cell.py

```diff
@@ -255,8 +256,7 @@ def _get_activation(self, F, inputs, activation, **kwargs):
             return F.Activation(inputs, act_type=activation, **kwargs)
```

Comment:
for string type, map the string to the most efficient operator. for example, if the string is 'tanh', instead of doing `F.Activation(act_type='tanh')`, do `F.tanh`, which doesn't require parsing the string at each call.